packages/yourloops/test/integration/assert/patient-list.assert.ts

```diff
@@ -755,10 +757,58 @@ const checkDefaultColumnsDisplay = () => {
+  expect(screen.getByRole('columnheader', { name: 'TIR' })).toBeVisible()
+  expect(screen.queryByRole('columnheader', { name: 'GMI' })).not.toBeInTheDocument()
   expect(screen.getByRole('columnheader', { name: 'Below range' })).toBeVisible()
-  expect(screen.queryByRole('columnheader', { name: 'Coefficient of Variation' })).not.toBeInTheDocument()
+  expect(screen.queryByRole('columnheader', { name: 'CV' })).not.toBeInTheDocument()
   expect(screen.getByRole('columnheader', { name: 'Last data update' })).toBeVisible()
   expect(screen.getByRole('columnheader', { name: 'Actions' })).toBeVisible()
 }
+
+const checkTooltipsColumnHeader = async (dataGridRows) => {
+  const tooltipTextMonitoringAlerts = 'Hover over the icons to learn more'
+  const tooltipTextMessages = 'Messages'
+  const tooltipTextBelowRange = 'Below Range'
+  const tooltipTextManagementGlucose = 'GMI (estimated HbA1c)'
+  const tooltipTextTimeInRange = 'Time in range'
+  const tooltipTextVariant = 'Coefficient of Variation'
+  const tooltipTextLastDataUpdate = 'Last data Update'
+
+  const monitoringAlertsColumnHeader = within(dataGridRows).getByText('Monitoring alerts')
+  const messagesColumnHeader = within(dataGridRows).getByText('Messages')
+  const TimeInRangeColumnHeader = within(dataGridRows).getByText('TIR')
+  const managementGlucoseColumnHeader = within(dataGridRows).getByText('GMI')
+  const belowRangeColumnHeader = within(dataGridRows).getByText('Below range')
+  const variantColumnHeader = within(dataGridRows).getByText('CV')
+  const LastDataUpdateColumnHeader = within(dataGridRows).getByText('Last data update')
+
+  await userEvent.hover(monitoringAlertsColumnHeader)
+  expect(await screen.findByText(tooltipTextMonitoringAlerts)).toBeVisible()
+  await userEvent.unhover(monitoringAlertsColumnHeader)
+
+  await userEvent.hover(messagesColumnHeader)
+  expect(await screen.findByText(tooltipTextMessages)).toBeVisible()
+  await userEvent.unhover(messagesColumnHeader)
+
+  await userEvent.hover(TimeInRangeColumnHeader)
+  expect(await screen.findByText(tooltipTextTimeInRange)).toBeVisible()
+  await userEvent.unhover(TimeInRangeColumnHeader)
+
+  await userEvent.hover(managementGlucoseColumnHeader)
+  expect(await screen.findByText(tooltipTextManagementGlucose)).toBeVisible()
+  await userEvent.unhover(managementGlucoseColumnHeader)
+
+  await userEvent.hover(belowRangeColumnHeader)
+  expect(await screen.findByText(tooltipTextBelowRange)).toBeVisible()
+  await userEvent.unhover(belowRangeColumnHeader)
+
+  await userEvent.hover(variantColumnHeader)
+  expect(await screen.findByText(tooltipTextVariant)).toBeVisible()
+  await userEvent.unhover(variantColumnHeader)
+
+  await userEvent.hover(LastDataUpdateColumnHeader)
+  expect(await screen.findByText(tooltipTextLastDataUpdate)).toBeVisible()
+  await userEvent.unhover(LastDataUpdateColumnHeader)
+
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏These 2 empty lines are not useful, they need to be removed